<launch>
  <arg name="pointcloud_map_path" />

  <node pkg="map_loader" exec="pointcloud_map_loader" name="pointcloud_map_loader" args="$(var pointcloud_map_path)">
    <remap from="output/pointcloud_map" to="/map/pointcloud_map" />
  </node>
</launch>
